server/apps/distributed-app/docs/modules/ROOT/pages/operate/webadmin.adoc

```diff
@@ -551,6 +551,58 @@ Response codes:
 * 400: The user is invalid
 * 404: The user is unknown
 
+=== Add a delegated user of a base user
```

Comment:
Maybe add a disclaimer explaining that delegation is only available on top of Cassandra products and not implemented yet on top of JPA backends.